base/special/exp2.jl

```diff
@@ -0,0 +1,446 @@
+# Copyright (c) 2005 David Schultz <das@FreeBSD.ORG>
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+# 1. Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+# 2. Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#
+# THIS SOFTWARE IS PROVIDED BY THE AUTHOR AND CONTRIBUTORS ``AS IS'' AND
+# ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED.  IN NO EVENT SHALL THE AUTHOR OR CONTRIBUTORS BE LIABLE
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS
+# OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+# HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+# LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+# OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
+# SUCH DAMAGE.
+
+const exp2f_table = [0.7071067811865476,
+0.7384130729697497,
+0.7711054127039704,
+0.8052451659746271,
+0.8408964152537145,
+0.8781260801866497,
+0.9170040432046712,
+0.9576032806985737,
+1.0,
+1.0442737824274138,
+1.0905077326652577,
+1.1387886347566916,
+1.189207115002721,
+1.241857812073484,
+1.2968395546510096,
+1.3542555469368927]
+
+#	exp2(z + eps)		eps	*/
+const exp2_combined_table = [0.707106781186592,   9.070522111187529e-14,
```

Comment:
Would it make any performance difference here to have an array of tuples here so that you load `(exp2, eps)` together?

Reply:
Ah, good call. You're totally right that it makes more sense (in Julia), and I don't think the splitting into two values would cause a problem.